src/util.ts

```diff
@@ -83,7 +83,7 @@ export class GoVersion {
 			this.isDevel = true;
 			this.commit = matchesDevel[0];
 		}
-		sendTelemetryEventForGoVersion(goVersion);
+		sendTelemetryEventForGoVersion(goVersion.format());
```

Comment:
The CI failure is real.

     Uncaught TypeError: Cannot read property 'format' of null
  	at new GoVersion (out/src/util.js:89:62)
  	at /Users/travis/gopath/src/github.com/microsoft/vscode-go/out/src/util.js:280:29
  	at ChildProcess.exithandler (child_process.js:294:7)
  	at maybeClose (internal/child_process.js:1021:16)
  	at Socket.<anonymous> (internal/child_process.js:430:11)
  	at Pipe.<anonymous> (net.js:586:12)

 Is "this.format" available by now?

Reply:
Fixed